tensorflow_addons/image/filters_test.py

```diff
@@ -94,11 +94,16 @@ def test_invalid_filter_shape(self):
         msg = ("The `filter_shape` argument must be a tuple of 2 integers.")
         image = tf.ones(shape=(1, 28, 28, 1))
 
-        for filter_shape in [(3, 3, 3), (3, None, 3), None]:
+        for filter_shape in [(3, 3, 3), (3, None, 3)]:
             with self.subTest(filter_shape=filter_shape):
                 with self.assertRaisesRegexp(ValueError, msg):
                     mean_filter2d(image, filter_shape=filter_shape)
 
+        filter_shape = None
+        with self.subTest(filter_shape=filter_shape):
+            with self.assertRaisesRegexp(TypeError, msg):
```

Comment:
seems kind of inconsistency, Tzu-Wei @WindQAQ

Reply:
I suppose that's why keras raise `ValueError` originally. Wonder if we should choose better engineering or standard.